resources/js/processes-catalogue/components/ProcessTab.vue

```diff
@@ -5,29 +5,359 @@
         title="My Requests"
         active
       >
-        <default-tab
-          :altText="$t('No Image')"
-          :titleText="$t('You have made no requests of this process.')"
-          :descriptionText="$t('All your requests will be shown here')"
-        />
+        <div v-if="showTabRequests">
+          <filter-table
+            :headers="tableHeadersRequests"
+            :data="dataRequests"
+            @table-row-click="(row) => handleRowClick('request', row)"
```

Comment:
use openRequest instead of handleRowClick

Reply:
method handleRowClick is necessary, using openRequest generates error